<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterPanel</class>
 <widget class="QWidget" name="FilterPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="gb_BoardType">
       <property name="title">
        <string>BoradType</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pb_Ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="3">
      <widget class="QDoubleSpinBox" name="dsp_delta"/>
     </item>
     <item row="3" column="3">
      <widget class="QSpinBox" name="sp_height"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Delta</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QDoubleSpinBox" name="dsp_sigmaspace"/>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="dsp_sigmay"/>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="dsp_sigmacolor">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>SigmaX</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>SigmaColor</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="dsp_scale"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="sp_width"/>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>SigmaY</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>SigmaSpace</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="sp_x"/>
     </item>
     <item row="4" column="3">
      <widget class="QSpinBox" name="sp_y"/>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="dsp_sigmax"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Scale</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Iterations</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="sp_iterations"/>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Ksize</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QSpinBox" name="sp_ksize"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
